src/main/resources/data/crimson/recipes/crafting_crimson_wood.json

```diff
@@ -0,0 +1,17 @@
+{
+	"type": "minecraft:crafting_shaped",
+	"group": "bark",
+	"pattern": [
```

Comment:
yuck tabs in json

Reply:
Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.

Reply:
for some reason it seems to be not showing up idk but I can see it exists in the actual repo